tools/cldr/cldr-to-icu/src/main/resources/ldml2icu_supplemental.txt

```diff
@@ -145,6 +140,69 @@
 //supplementalData/calendarData/calendar[@type="(%W)"]/eras/era[@type="(%W)"][@(start|end)="(%A)"]
     ; /calendarData/$1/eras/$2/$3:intvector ; values=&ymd($4)
 
+# units.xml
+
+//supplementalData/unitConstants/unitConstant[@constant="(%W)"][@value="(%A)"]
+    ; /unitConstants/$1 ; values="$2"
+
+//supplementalData/unitConstants/unitConstant[@constant="(%W)"][@value="(%A)"][@status="(%W)"]
```

Comment:
use (?:...)? for optional attributes you just ignore.
I.e. you can roll up this rule into the parent one.

Reply:
Done, PTAL?